src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Triggers/ServiceBusTriggerAttributeBindingProvider.cs

```diff
@@ -74,6 +74,7 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
                 throw new InvalidOperationException(string.Format(CultureInfo.CurrentCulture, "Can't bind ServiceBusTrigger to type '{0}'.", parameter.ParameterType));
             }
 
+            attribute.Connection = Resolve(attribute.Connection);
```

Comment:
Problem also exists in ServiceBusAttributeBindingProvider

Reply:
I added connection resolving to  `ServiceBusTriggerAttributeBindingProvider`, `ServiceBusAttributeBindingProvider `and `EventHubTriggerAttributeBindingProvider`. There is no such thing as `EventHubAttributeBindingProvider`, I believe `INameResolver `is not used in `EventHub `output binding [at all](https://github.com/Azure/azure-webjobs-sdk/blob/b7248b21998e46ffa34529b7126acde34a89bb4f/src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubExtensionConfigProvider.cs#L72).


`StorageAccountProvider `resolves connection for all storage bindings.